regression/pages/common/utils.py

```diff
@@ -0,0 +1,526 @@
+    """
+    A helper function to fill multiple fields in a form
+    Args:
+        page:
+        css_selectors:
+        values:
+    """
+    for css_selector, value in izip(css_selectors, values):
+        page.q(css=css_selector).fill(value)
+
+
+def select_value_from_drop_down(page, drop_down_name, value):
+    """
+    A helper function to select value from drop down
+    Args:
+        page:
+        drop_down_name:
+        value:
+    """
+    page.wait_for_element_visibility(
+        'select[name=' + drop_down_name + ']', 'Drop down is visible')
+    page.q(
+        css='select[name=' + drop_down_name + '] option[value="{}"]'.format(
+            value)).click()
+    page.wait_for(
+        lambda: page.q(
+            css='select[name=' + drop_down_name + '] option[value="{}"]'.
+            format(value)).selected,
+        "Correct value is selected",
+        timeout=DEFAULT_TIMEOUT
+    )
+
+
+def select_values_from_drop_downs(page, drop_down_names, values):
+    """
+    A helper function to select values from multiple drop downs on a page
+    Args:
+        page:
+        drop_down_names:
+        values:
+    """
+    for drop_down_name, value in izip(
+            drop_down_names, values
+    ):  # pylint: disable=cell-var-from-loop
+        page.wait_for_element_visibility(
+            'select[name=' + drop_down_name + ']', 'Drop down is visible')
+        page.q(
+            css='select[name=' + drop_down_name + '] option[value="{}"]'.
+            format(value)).click()
+        page.wait_for(
+            lambda: page.q(
+                css='select[name=' + drop_down_name + '] option[value="{}"]'.
+                format(value)).selected,
+            "Correct value is selected",
+            timeout=DEFAULT_TIMEOUT
+        )
+
+
+def click_checkbox(page, css_selector):
+    """
+    A helper function to click check boxes
+    Args:
+       page:
+       css_selector:
+    """
+    # pylint: disable=unnecessary-lambda
+    page.wait_for_element_visibility(css_selector, 'wait for target checkbox')
+    page_elem = page.q(css=css_selector).results[0]
+    if not page_elem.is_selected():
+        page.browser.execute_script("arguments[0].click();", page_elem)
+        page.wait_for(
+            lambda: page_elem.is_selected(),
+            'Target checkbox is selected'
+        )
+
+
+def remove_spaces_from_list_elements(list_with_spaces):
+    """
+    A helper function to remove spaces from all items in the list
+    Args:
+      list_with_spaces:
+    """
+    return [x.replace(' ', '') for x in list_with_spaces]
+
+
+def get_text(page, css_selectors):
+    """
+    A helper function to get text values for a list of web elements
+    Args:
+        page:
+        css_selectors:
+    Returns:
+        list_of_text:
+    """
+    list_of_text = []
+    for css_selector in css_selectors:
+        list_of_text.append(page.q(css=css_selector).text[0])
+    return list_of_text
+
+
+def convert_date_format(target_date):
```

Comment:
This function also needs to be renamed to make it clear that it's putting a date into a particular format. when I see `convert_date_format` I think I can take any date and convert it to any other format. That's not true...this is a very specific function.